nntrainer/tensor/tensor.cpp

```diff
@@ -213,14 +241,18 @@ int Tensor::divide_i(float const &value) {
   return this->multiply_i(1.0f / value);
 }
 
-Tensor Tensor::divide(float const &value) {
+Tensor Tensor::divide(float const &value) const {
   if (value == 0.0f) {
     throw std::runtime_error("Error: Divide by zero");
   }
 
   CLONE_OP_I(divide_i, value);
 }
 
+Tensor Tensor::divide(float const &value, Tensor &out) const {
+  throw nntrainer::exception::not_supported("NYI");
```

Comment:
can you add this same as `multiply` with the same interface?

Reply:
you mean throw?  it is implemented at following commit :)